tools/src/main/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommandOptions.java

```diff
@@ -83,6 +84,13 @@ public ReassignPartitionsCommandOptions(String[] args) {
             .withRequiredArg()
             .describedAs("brokerlist")
             .ofType(String.class);
+
+        bootstrapControllerOpt = parser.accepts("bootstrap-controller", "The controller to use for reassignment. " +
+                        "By default, the tool will get the controller from the broker.")
```

Comment:
`the tool will get the controller from the broke` this description is not correct. The controller is "quorum controller" rather than "broker controller"